pyccel/codegen/printing/fcode.py

```diff
@@ -741,6 +741,29 @@ def _print_NumpyFloor(self, expr):
         result_code = self._print_MathFloor(expr)
         return 'real({}, {})'.format(result_code, self.print_kind(expr))
 
+    def _print_NumpyArange(self, expr):
+        start  = self._print(expr.start)
+        step   = self._print(expr.step)
+        shape  = PyccelMinus(expr.shape[0], LiteralInteger(1))
+        target = Variable(NativeInteger(), name =  self.parser.get_new_name('i'))
```

Comment:
Why do you call this variable `target`? Is it not an index?

Reply:
i just followed the same naming we do in For printer

Reply:
I don't know when the `For` printer was written, but I doubt that it is a model of style.